runtimes/pytorch/ubi8-python-3.8/Pipfile

```diff
@@ -12,9 +12,9 @@ name = "pytorch"
 
 [packages]
 # PyTorch packages
-tensorboard = "~=2.14.0"
-torch = {version = "~=2.0.0", index = "pytorch"}
-torchvision = {version = "~=0.15.2", index = "pytorch"}
+tensorboard = ">=2.14.0"
+torch = {version = "~=2.2.2", index = "pytorch"}
+torchvision = {version = ">=0.15.2", index = "pytorch"}
```

Comment:
Is it correct to replace `~=` with `>=`? Can't you leave ~= and increase the number after it? https://peps.python.org/pep-0440/#version-specifiers (because I always forget what these mean)

Reply:
Yes i will fix it, i was checking something. I will ping you again when i will finish the work :slightly_smiling_face:

Reply:
Alright all set!